templates/index.html

```diff
@@ -2,10 +2,11 @@
 
 {% block content %}
 {% if current_user.is_authenticated %}
-    <div class='some-page-wrapper'>
+    <div class='{{ config["DASHBOARD_PAGE_WRAPPER"] }}'>
```

Comment:
dont do this, change the css file used instead

Reply:
i could create two css and change the names of the field to be the same from boht files but different colors.

Reply:
two different css based on might/dark themes